src/script/ceph_backport.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -ex
+#
+# ceph_backport.sh
+#
+# Ceph backporting script
+#
+# Assumes you have forked ceph/ceph.git, cloned your fork, and are running the
+# script in the local clone!
+#
+# With this script, and assuming you are working on backport issue
+# http://tracker.ceph.com/issues/19206 (which is a jewel backport), backporting
+# workflow becomes something like this:
+#
+# git remote add ceph http://github.com/ceph/ceph.git
+# git fetch ceph
+# git checkout -b wip-19206-jewel ceph/jewel
+# git cherry-pick -x $MASTER_COMMIT_SHA1
+# # rinse and repeat, resolve conflicts, describe conflicts and how they
+# # were resolved in the commit message . . .
+# ceph_backport.sh jewel 19206
+#
+# The script takes care of opening the backport PR and updating the tracker issue #19206 
+#
+# However, before you start you will need to find the right values for
+# the following:
+#
+#redmine_key=3432d438a34c382022115438a34c38202213bd03
+#redmine_user_id=1357 # as found in the URL near "Logged in as " top right of each page
+#github_token=438a34c382022138a34c3820221152d438a3da39
+#
+# The actual values are set in a separate file which you have to create.
+# This script just reads in the values, like so:
+source $HOME/bin/backport_common.sh
+
+# this will need tweaking if your github username != $USER
+github_user=$USER
+
+# check number of arguments passed - should be two (milestone and backport
+# tracker issue number)
+if [ "$#" -ne 2 ]; then
+    echo "Usage:"
+    echo "   $0 [MILESTONE] [BACKPORT_TRACKER_ISSUE_NUMBER]"
+    echo
+    echo "Example:"
+    echo "   $0 jewel 19206"
+    echo
+    echo "Must be run from inside the local git clone"
+    exit 1
+fi
+milestone=$1
```

Comment:
can we make milestone as env variable or so? Then there is no need to set it while doing a bunch of backports